hangman.js

```diff
@@ -1 +1,28 @@
-console.log("welcome to hangman");
\ No newline at end of file
+console.log("welcome to hangman");
+
+var wordBank= ['monday', 'ennui', 'legit', 'beer', 'butt', 'llama', 'wombat'];
+var answer;
+var gameWord;
+var correct_guesses= [];
+var incorrect_guesses= [];
+
+
+function chooseWord(){
+  var wordIndex = Math.floor(Math.random()*wordBank.length);
+  var hangWord = wordBank[wordIndex];
+
+  return hangWord;
+}
+
+hangWord = chooseWord();
```

Comment:
You declared `hangWord` inside the function `chooseWord`. It therefore exists only inside `chooseWord`. The `hangWord` here is a different variable. As such, it should be declared with `var`.